### DISCUSSION TITLE
How can i add a runnable to RustLsp runnable

### DISCUSSION THREAD

**Discussion by NightProg** [mrcjkb/rustaceanvim#439] [Q&A] [answered]
Hello, I am asking how can I add a RustLsp runnable I search, but I didn't find anything. ( I want to add the cargo run runnable) 

Thanks for your help

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

rustaceanvim's `:RustLsp runnables` command queries rust-analyzer. 

You can refer to [the rust-analyzer manual](https://rust-analyzer.github.io/manual.html) for configuration options.

**Reply by NightProg** [reply]
Thanks you i will look at that
